Add new terms for glossary: RHEL software management (#261)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

*See also*:

[discrete]
[[dnf-automatic]]
==== image:images/yes.png[yes] DNF Automatic (noun)

> *Description*: Use _DNF Automatic_ to refer to a Red Hat Enterprise Linux command-line interface suited for automatic and regular package updates.

Reviewer: ```suggestion
*Description*: An alternative to the DNF package manager that's optimized for automatic, regular updates.
```
